html/partials/50x.shtml

```diff
@@ -1,2 +1,3 @@
-<p>Sorry, the page you are looking for is currently unavailable.</p>
-<p>Please try again later.</p>
+<h2>An error occurred</h2>
+<p>The page you’re looking for isn’t available.</p>
```

Comment:
We think: the content should be the same as on the 500 page so "There's a problem. Please try again in a few moments."